### PR TITLE
chore(client): add Arabic i18n json files

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Comment by naomi-lgbt** [comment]
Blocked until #48127 lands.

Then, we need to let the client UI download run, and rebase this PR to not change files that are managed through Crowdin.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
To clarify, the `intro.json` file should not have been copied over. The `links.json` file should, instead.

**Review by naomi-lgbt — APPROVED** [review]
I was wrong this doesn't need to be blocked.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**
